extensions/wikia/AdEngine/js/lookup/prebid/prebid.js

```diff
@@ -54,7 +54,27 @@ define('ext.wikia.adEngine.lookup.prebid', [
 				return;
 			}
 
-			log('User opt-in for prebid', log.levels.info, logGroup);
+			win.__cmp = function(command, version, callback) {
```

Comment:
We can define this function out of `call` function scope, am I right?

Reply:
No, because we need optIn variable which is available here. Also, we don't need it if prebid is not enabled = no call will be performed.